WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -568,8 +568,19 @@ - (void)showOnboardingTips
 
 #pragma mark - Actions
 
-- (void)showBlogSelectorPrompt
+- (void)showBlogSelectorPrompt:(WPBlogSelectorButton*)sender
 {
+    // If we are editing an existing post, we need to simply exit
+    // the screen we are currently on and return.
```

Comment:
These comments are documenting a not-so-obvious behaviour of the method.  I think it would be better to include them in the `@details` comment tag at the head of the method definition.